source/connect/favorite-connections/import-export-cli/import.txt

```diff
@@ -12,4 +12,153 @@ Import Favorite Connections with the CLI
+executable:
+
+.. code-block:: sh
+
+   ./MongoDB\ Compass --import-connections=/tmp/compass-connections/favorites.json
+   
+.. note::
+
+   The name and file path of the executable depend on your operating
+   system. The preceding command is for macOS.
+   
+Results
+```````
+
+After you run the import command, you will see this output:
+
+.. code-block:: sh
+   :copyable: false
+
+   Importing connections from "/tmp/compass-connections/favorites.json" (without passphrase)
+   
+MongoDB Compass starts and you will see the imported connections under
+:guilabel:`Saved Connections` on the Connect screen.
+
+Import Encrypted Favorite Connections
+-------------------------------------
+
+Use this procedure to import a list of favorite connections that is
+:ref:`encrypted with a passphrase <compass-encrypt-connections-cli>`.
+
+Procedure
+~~~~~~~~~
+
+To import unencrypted favorite connections with the |compass-short|
+:abbr:`CLI (Command-Line Interface)`, specify:
+
+- The path to the |compass| executable. The name and filepath of the
+  executable depend on your operating system.
+
+- The ``--import-connections`` option set to the destination of the file
+  containing favorite connections.
+
+- The ``--passphrase`` option set to the passphrase used to encrypt the
+  exported file.
+
+Your operation should resemble the following prototype:
+
+.. code-block:: sh
+
+   <path-to-Compass-executable> \
+   --import-connections=<filename> \
+   --passphrase=<passphrase>
+
+Example
+~~~~~~~
+
+The following example imports favorite |compass-short| connections from
+a file with the path
+``/tmp/compass-connections/favorites-encrypted.json`` that is encrypted
+with the passphrase ``abc123``.
+
+Run the following command in the folder containing your MongoDB Compass
+executable:
+
+.. code-block:: sh
+
+   ./MongoDB\ Compass \
+   --import-connections=/tmp/compass-connections/favorites-encrypted.json \
+   --passphrase=abc123
+
+.. note::
+
+   The name and file path of the executable depend on your operating
+   system. The preceding command is for macOS.
+
+Results
+```````
+
+After you run the import command, you will see this output:
+
+.. code-block:: sh
+   :copyable: false
+
+   Importing connections from "/tmp/compass-connections/favorites-encrypted.json" (with passphrase)
+   
+MongoDB Compass starts and you will see the imported connections under
+:guilabel:`Saved Connections` on the Connect screen.
+
+Errors
+~~~~~~
+
+When you try to import encrypted favorite connections, you may
+see these errors:
+
+- If do not specify a passphrase, you will see this error:
+
+  .. code-block:: sh
+     :copyable: false
+
+     Failed to perform operation Be [Error]: Input file contains encrypted
+     secrets but no passphrase was provided
+   
+- If you specify an incorrect passphrase, you will see this error:
+
+  .. code-block:: sh
+     :copyable: false
+  
+     Failed to perform operation Be [Error]: Cannot decrypt due to corrupt
```

Comment:
```suggestion
     Failed to perform operation [Error]: Cannot decrypt due to corrupt
```
What's this `Be` from? Maybe we can remove it.
Same with the `Failed to perform operation Be [Error]: Input file contains encrypted
     secrets but no passphrase was provided` error message above.

Reply:
@Anemy, the "Be" was in the error output itself. Here is the full output I see in my terminal when trying to import with a bad passphrase:

> \> jeff@M-C02D56EEMD6M MacOS % ./MongoDB\ Compass\ Beta --import-connections=/tmp/compass-connections/favorites-encrypted.json --passphrase=abc124

> Importing connections from "/tmp/compass-connections/favorites-encrypted.json" (with passphrase)
(node:29717) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `MongoDB Compass Beta --trace-deprecation ...` to show where the warning was created)
Failed to perform operation Be [Error]: Cannot decrypt due to corrupt data or wrong passphrase

Not sure if the "Be" has something to do with "Beta", since I'm trying this on Compass Beta? I can remove it from the docs if it shouldn't be there.

Reply:
Huh, fine either way I think. Since it's in the output it might make seo a bit better? It does read a bit funnier though.